src/spac/data_utils.py

```diff
@@ -506,9 +503,23 @@ def select_values(data, annotation, values=None):
         values = [values]
 
     if isinstance(data, pd.DataFrame):
+        # Check if the annotation exists in the DataFrame
+        if annotation not in data.columns:

```

Comment:
Can we use the check list in list utils instead? This way, we have one place to edit our error messages.